Modules/Filtering/BiasCorrection/include/itkN4BiasFieldCorrectionImageFilter.h

```diff
@@ -428,6 +428,9 @@ class N4BiasFieldCorrectionImageFilter :
   ArrayType    m_NumberOfControlPoints;
   ArrayType    m_NumberOfFittingLevels;
 
+  // Number of pixels of the input image that are included with the filter.
+  std::size_t m_NumberOfIncludedPixels{ 0 };
```

Comment:
From github review it appears that there is no need for this to be member variable.  It could be a local variable.  Perhaps I overlooked where this is needed across function boundaries.

Can you please confirm in your local builds.

Reply:
@hjmjohnson Thanks for your interesting remark. Unfortunately there currently _is_ a function boundary between the estimation of `m_NumberOfIncludedPixels`, in `GenerateData()` and its use, in the private helper function `UpdateBiasFieldEstimate(RealImageType*)`. However, `GenerateData()` could pass this number as an extra parameter to `UpdateBiasFieldEstimate`. I'll have another look!

Reply:
Please do not merge: I still would like to have another look!

Reply:
@hjmjohnson Please check how I processed your comment: I removed the member variable `m_NumberOfIncludedPixels` from the initial version of the PR, and added a local variable `numberOfIncludedPixels` to `GenerateData()`, that is passed as parameter to `UpdateBiasFieldEstimate`